File: 2141019334/OSW_2141019334/Concepts/Basics/q7.c
#include <stdio.h>
int main()
{
    int a, b, c;
    printf("enter number a ");
    printf("enter number b ");
    printf("enter number c ");
    scanf("%d %d %d", &a, &b, &c);
    int sum = a + b + c;
    printf("sum=%d", sum);
}

File: Java Programs/Arrays/q7.java
import java.util.*;
class q7
{
    static void main()
    {
        Scanner in=new Scanner(System.in);
        int a[]=new int[5];
        int b[]=new int[5];
        int c[]=new int[5];
        int i,j,k;
        for(i=0;i<=4;i++)
        {
            System.out.println("Enter Number.");
            a[i]=in.nextInt();
            for(k=4;k>-1;k--)
            {
                System.out.println("Enter Number.");
                b[k]=in.nextInt();
                c[i]=a[i]+b[k];
            }
        }
        for(j=0;j<=4;j++)
        {
            System.out.println("The Sum of the 2 elements is="+c[j]);
        }
    }
}